cmd/up/activate.go

```diff
@@ -119,7 +126,9 @@ func (up *upContext) activate() error {
 	}
 
 	if up.isRetry {
-		analytics.TrackReconnect(true)
+		// isDevPodRecreated if the pod UID has changed from last retry
+		isDevPodRecreated := lastPodUID != up.Pod.UID
```

Comment:
@pchico83 is a boolean sufficient or would a string indicating the cause look better? If we could not detect the cause it would be an `unrecognized` until we can detect that type.
For example:

```json
{
  "cause": "unrecognised"/"dev-pod-recreated"/"lost-connection"
}
```

Reply:
Yes, a string is more flexible for future changes